docs/usage/self-hosted-configuration.md

```diff
@@ -725,6 +733,14 @@ This is currently applicable to `npm` and `lerna`/`npm` only, and only used in c
 
 ## token
 
+## translationsFilePath
+
+This option is specific the PO(Portable Object) file's path.
+
+Renovate uses the default single domain "messages" for i18n.
+
+You can translate your own edition by inovking `msginit` program, and then invoke `msgmerge` program to add more `msgid`.
```

Comment:
I think that this needs to link to a fuller document about how to create own translations, otherwise one sentence is too little